src/modules/cmdpal/ext/Microsoft.CmdPal.Ext.TimeDate/Helpers/AvailableResultsList.cs

```diff
@@ -109,8 +110,9 @@ internal static List<AvailableResult> GetList(bool isKeywordSearch, SettingsMana
                         {
                             value = dtObject.ToString(value, CultureInfo.CurrentCulture);
                         }
-                        catch
+                        catch (Exception ex)
                         {
+                            Logger.LogError($"Unable to format date time with format: {value}. Error: {ex.Message}");
```

Comment:
please move inside the if. we can get false positives if the `.ToString()` can't replace anything and we have custom formats to replace.